packages/rest/src/keys.ts

```diff
@@ -85,10 +86,32 @@ export namespace RestBindings {
     'rest.errorWriterOptions',
   );
 
+  /**
+   * Binding key for request body parser options
+   */
   export const REQUEST_BODY_PARSER_OPTIONS = BindingKey.create<
     RequestBodyParserOptions
   >('rest.requestBodyParserOptions');
 
+  /**
+   * Binding key for request body parser
+   */
+  export const REQUEST_BODY_PARSER = BindingKey.create<RequestBodyParser>(
+    'rest.requestBodyParser',
+  );
+
+  export const REQUEST_BODY_PARSER_JSON = BindingKey.create<BodyParser>(
```

Comment:
Is it a good idea to turn binding keys of individual parsers into a public API? Personally, I would expect that these keys should be treated as an implementation detail, similarly to how we are not making much guarantees about the binding keys for controllers, repositories and routes.

Reply:
@raymondfeng you haven't addressed nor responded to this comment. PTAL again.

Reply:
Let's discuss this point before landing the pull request. If we wanted to remove these public keys later, then we would have to wait until we are ready for a semver-major version.